.github/workflows/PackageCI.yml

```diff
@@ -0,0 +1,23 @@
+name: "Oscar CI (package tests)"
```

Comment:
Besides the name, this file looks identical to the previous one ("FullCI.yml"), did I miss something?

Reply:
This is a copy and paste error in `FullCI.yml`. Basically, I've copied the block without adjusting the `test` parameter.

It should be fixed now.